src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -62,6 +65,23 @@ goog.require('ga_permalink');
               return coord + ' ' + zone;
             };
 
+            var updateW3W = function() {
+              $http.get(w3wUrl, {
+                params: {
+                  key: w3wKey,
+                  position: coord4326[1] + ',' + coord4326[0],
+                  lang: gaLang.get()
+                }
+              }).success(function(response) {
+                var words = response.words;
+                if (words && words.length == 3) {
+                  scope.w3w = response.words[0] + '.' +
+                              response.words[1] + '.' +
+                              response.words[2];
+                }
```

Comment:
We should also manage the case the response conatins an error message like RM : {"error":41,"message":"Invalid language"}`` 

May be displaying  a ` - ` or nothing.